tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "allowJs": true,
+    "declaration": false,
+    "skipLibCheck": true
```

Comment:
I needed this because I was getting tonnes of type errors in `@google-cloud/common-grpc` type definitions. @JustinBeckwith any ideas?

```
> tsc -p . && cp src/v2/*.json build/src/v2 && cp -r protos build

node_modules/@google-cloud/common-grpc/build/src/service.d.ts:210:5 - error TS2416: Property 'request' in type 'GrpcService' is not assignable to the same property inbase type 'Service'.
  Type '(protoOpts: any, reqOpts: any, callback: any) => any' is not assignable to type '{ (reqOpts: DecorateRequestOptions): Promise<Response>; (reqOpts: DecorateRequestOptions, callback: BodyResponseCallback): void; }'.

210     request(protoOpts: any, reqOpts: any, callback: any): any;
        ~~~~~~~

node_modules/@google-cloud/common-grpc/build/src/service.d.ts:221:5 - error TS2416: Property 'requestStream' in type 'GrpcService' is not assignable to the same property in base type 'Service'.
  Type '(protoOpts: any, reqOpts: any) => any' is not assignable to type '(reqOpts: DecorateRequestOptions) => Request'.

221     requestStream(protoOpts: any, reqOpts: any): any;
```

Reply:
that's weird *scratches head* lemme take a look